.gitpod.Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM gitpod/workspace-full:latest
+
+USER root
+
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
+    #
+    # Verify git, process tools, lsb-release (common in install instructions for CLIs) installed
+    && apt-get -y install git iproute2 procps iproute2 lsb-release \
+    #
+    # Install C compilers (gcc not enough, so just went with build-essential which admittedly might be overkill),
```

Comment:
If you can get conda to work then can just install the compiler from conda-forge

https://anaconda.org/conda-forge/c-compiler

Reply:
This is copied from the existing Dockerfile, so I thought it would be ok?